extensions/blocks/eventbrite/eventbrite.php

```diff
@@ -46,15 +66,61 @@ function render_block( $attr, $content ) {
 		true
 	);
 
+	// Show the embedded version.
+	if ( empty( $attr['useModal'] ) && ( empty( $attr['style'] ) || 'modal' !== $attr['style'] ) ) {
+		return render_embed_block( $attr );
+	} else {
+		return render_modal_block( $attr, $content );
+	}
+}
+
+/**
+ * Render block with embed style.
+ *
+ * @param array $attr Eventbrite block attributes.
+ * @return string Rendered block.
+ */
+function render_embed_block( $attr ) {
 	$widget_id = wp_unique_id( 'eventbrite-widget-' );
 
-	wp_enqueue_script( 'eventbrite-widget', 'https://www.eventbrite.com/static/widgets/eb_widgets.js', array(), JETPACK__VERSION, true );
+	$is_amp = Jetpack_AMP_Support::is_amp_request();
 
-	// Add CSS to hide direct link.
-	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
+	// $content contains a fallback link to the event that's saved in the post_content.
+	// Append a div that will hold the iframe embed created by the Eventbrite widget.js.
+	$classes = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attr );
+
+	$classes .= ' wp-block-jetpack-eventbrite--embed';
+
+	$direct_link = sprintf(
+		'<a href="%s" rel="noopener noreferrer" target="_blank" class="eventbrite__direct-link" %s>%s</a>',
+		esc_url( $attr['url'] ),
+		$is_amp ? 'placeholder fallback' : '',
+		esc_html__( 'Register on Eventbrite', 'jetpack' ),
+	);
+
+	if ( $is_amp ) {
+		$embed = sprintf(
+			'<amp-iframe src="%s" layout="responsive" resizable width="1" height="1" sandbox="allow-scripts allow-same-origin allow-forms"><button overflow>%s</button>%s</amp-iframe>',
```

Comment:
Note that the `resizable` attribute doesn't currently do anything. Ideally Eventbrite would look for `#amp=1` in the loaded URL and opt to send AMP's resize messages.